### PR TITLE
Change /ban & /roomban alias

### PR DESCRIPTION
**Pull request by Bryan-0**
- Zarel commented in #2607 that both versions of /ban should probably be default to roomban, making /ban = to /globalban.

### CODE REVIEW

**Comment by panpawn** [comment]
meh, I still don't see people mixing up these commands in the first place

**Comment by tr12345** [comment]
Honestly I don't think its worth that, primarily because there isn't such a command called /roomlock, its just global lock. It looks fine the way it is imo

**Comment by AustinXII** [comment]
Seems like a haste for all staff to get used to typing /b or /ban instead of /rb for room bans and yeah agree on what @panpawn said

**Comment by Zarel** [comment]
It's not so much about staff mixing the command up, more that roombans are used more than globalbans, so it should be easier to type

File: commands.js [hunk 1]
```diff
@@ -978,9 +978,9 @@ exports.commands = {
 		connection.popup(buffer.join("\n\n"));
 	},
 
-	rb: 'roomban',
-	roomban: function (target, room, user, connection) {
-		if (!target) return this.parse('/help roomban');
+	b: 'ban',
```
Comment (Zarel) [inline commands.js]: We can keep `/rb` and `/roomban` as aliases.

**Comment by kotarou3** [comment]
`/rub` = `/roomunban` when

**Comment by Slayer95** [comment]
lol please disregard the /rub comment, @Bryan-0

**Comment by Bryan-0** [comment]
hmm

File: commands.js [hunk 2]
```diff
@@ -1405,10 +1407,11 @@ exports.commands = {
 		this.globalModlog("BAN", targetUser, " by " + user.name + (target ? ": " + target : ""));
 		return true;
 	},
-	banhelp: ["/ban OR /b [username], [reason] - Kick user from all rooms and ban user's IP address with reason. Requires: @ * & ~"],
+	globalbanhelp: ["/globalban OR /gb [username], [reason] - Kick user from all rooms and ban user's IP address with reason. Requires: @ * & ~"],
 
-	unban: function (target, room, user) {
-		if (!target) return this.parse('/help unban');
+	ungban: 'unglobalban',
```
Comment (Zarel) [inline commands.js]: The commands being `gb` and `ungban` seem kinda weird.

Comment (Slayer95) [inline commands.js]: Yea. We probably don't need any alias at all for global bans.

Comment (Bryan-0) [inline commands.js]: So, remove ``/gb`` and ``/ungban``?

Comment (panpawn) [inline commands.js]: Yea, sounds like it

Comment (Zarel) [inline commands.js]: Keep `/gb`, remove `/ungban`

**Comment by Bryan-0** [comment]
Done. @Zarel

**Comment by Bryan-0** [comment]
https://github.com/Zarel/Pokemon-Showdown/commit/05ae323fcddf97d865ec5e588d8db26e06b04041

**Pull request closed without merging**